atomic_reactor/schemas/user_params.json

```diff
@@ -31,6 +31,26 @@
     "koji_task_id": {"type": "integer"},
     "koji_upload_dir": {"type": "string"},
     "name": {"type": "string"},
+    "parent_images_digests": {
+      "type": "object",
+      "patternProperties": {
+        "^.+:.+": {
```

Comment:
Maybe include `$` at the end of the regex?

Reply:
Added